pjnath/src/pjnath/ice_session.c

```diff
@@ -3734,6 +3746,29 @@ PJ_DEF(pj_status_t) pj_ice_sess_on_rx_pkt(pj_ice_sess *ice,
 
         PJ_RACE_ME(5);
 
+        if (ice->opt.check_src_addr) {
+            for (i = 0; i < ice->rcand_cnt; ++i) {
```

Comment:
Won't this slow down ICE tp considerably, if we have to iterate through all remote candidates for every packet?